src/main/java/net/glowstone/constants/GlowParticle.java

```diff
@@ -101,13 +99,15 @@
             return new Float[]{red, green, blue, size};
         });
         registerParticleDataHandler(ItemStack.class, PARTICLE_HANDLER_RETURN_SELF);
-        registerParticleDataHandler(BlockData.class, o -> {
+        // TODO: Figure out what the correct result for this is, since not all BlockData can be converted to a single
```

Comment:
I thought every BlockData had an integer ID?

Reply:
Fixed it in the latest commit.